build.cmd

```diff
@@ -6,8 +6,26 @@ setlocal
 ::       means that that rebuilding cannot successfully delete the task
 ::       assembly. 
 
-echo %2 | find /I "outerloop"
-if "%errorlevel%" equ "0" (
+set outloop=false
+
+:START_CMDLINE_PARSE
+set SWITCH=%1
+if {%SWITCH%} == {} goto :END_CMDLINE_PARSE
+set VALUE=%2
+
+if /i "%SWITCH%" == "/p:WithCategories" (
```

Comment:
The script engineer apparently treat '=' as a command line parameter separator as well. If you pass in '/p:WithCategories=OuterLoop', %1 will be /p:WithCategories, %2 will be OuterLoop.

I have verified it work with '/p:WithCategories=OuterLoop'.